### ISSUE TITLE
Astronvim exits when I try to close a currently viewing file by clicking on 'x' icon on the file tab

### ISSUE THREAD

**Issue by vikramreddym** [AstroNvim/AstroNvim#1210]
### Question

Hi,

After I open multiple files, when I try to close a file (currently viewing) by clicking on 'x' icon on the file tab using mouse, nvim completely exits instead of just closing only that particular file. I am using nvim v0.8.0 and astronvim latest stable. Is this normal behaviour?

Thanks

### Steps to reproduce:
nvim == v0.8.0
Astronvim latest at the time of writing this post.

1) Start nvim and open multiple files from the explorer.
2) click on the x for the most recently opened file which is also in focus (i.e. currently viewing).
![Screen Shot 2022-10-31 at 2 33 45 PM](https://user-images.githubusercontent.com/10250262/199084072-aa257840-9433-42ed-932c-a9b3edd776c7.png)
3) nvim will completely exit instead of just closing the particular file.

**Comment by derekmcloughlin** [comment]
Same happening to me.

**Comment by vikramreddym** [comment]
> Could you please share screenshots and the exact steps for reproducing this?

Added more details in the description.

**Comment by mehalter** [comment]
Ah I can replicate this. Thanks for the extra information @vikramreddym ! I will investigate and try to get a fix out :) sorry!

**Comment by jayp0521** [comment]
This fixed it for me

`user.plugins.bufferline`
```
return {
	options = {
		close_command = function(bufnum)
			require("bufdelete").bufdelete(bufnum, true)
		end,
	},
}
```

**Comment by mehalter** [comment]
Ah this is the exact option I was hoping to find! Thanks @jayp0521 ! I'll make a bug fix with this tomorrow :)

**Comment by mehalter** [comment]
@vikramreddym I have pushed a fix for this to `nightly` (PR https://github.com/AstroNvim/AstroNvim/pull/1218). Could you please check out that version and see if it resolves this problem? If so, I can go ahead and tag a new patch release :)

**Comment by mehalter** [comment]
This should be resolved now with v2.5.1 !
